cmd/main.go

```diff
@@ -52,6 +53,23 @@ func init() {
 	}
 }
 
+func StartSecurityPolicyController(mgr ctrl.Manager, service common.Service) {
+	securityReconcile := &controllers.SecurityPolicyReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+	}
+	if securityService, err := securitypolicy.InitializeSecurityPolicy(service); err != nil {
+		log.Error(err, "failed to initialize securitypolicy service", "controller", "SecurityPolicy")
+		os.Exit(1)
+	} else {
+		securityReconcile.Service = securityService
+	}
+	if err := securityReconcile.Start(mgr); err != nil {
+		log.Error(err, "failed to create controller", "controller", "SecurityPolicy")
```

Comment:
follow line 62 style, "failed to create securitypolicy controller",  do we still need ' "controller", "SecurityPolicy"'?

Reply:
I think so, If another controller comes in, we also log like this, so we know which controller fail.